setup.cfg

```diff
@@ -56,7 +56,7 @@ max_line_length = 88
 convention = google
 
 [tool:isort]
```

Comment:
Please could you replace all these isort settings with `profile = "black"` and move it into `pyproject.toml`?

Reply:
moved all the configs I can into pyproject.toml